doc/api/next_api_changes/development/20391-AG.rst

```diff
@@ -0,0 +1,8 @@
+fontTools for type 42 subsetting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new dependency known as `fontTools <https://fonttools.readthedocs.io/>`_
+is integrated in with Maptlotlib 3.5
```

Comment:
```suggestion
is now required for Maptlotlib 3.5.
```

Reply:
I think  the wording "is integrated in with" could be read to mean that we ship a vendored copy of the library.